lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -14,10 +14,6 @@
 charmcraft fetch-lib charms.tls_certificates_interface.v2.tls_certificates
 ```
 
-Add the following libraries to the charm's `requirements.txt` file:
-- jsonschema
-- cryptography
-
 Add the following section to the charm's `charmcraft.yaml` file:
```

Comment:
```suggestion
This library depends on the `cryptography` and `jsonschema` libraries.
To pack a charm with this library, you need to updated its `charmcraft.yaml` file
to either use binary packages:

          parts:
            charm:
              charm-binary-python-packages: [cryptography, jsonschema]
              
or install necessary dependencies to build from source:
```

Reply:
I imagine this doc fix would be sufficient to close #44, @gruyaume?

Reply:
Do we really want to encourage this? It's a nice hack for development, but production builds should be building from source right?